tools/azsdk-cli/docs/specs/99-repo-tooling-contract.md

```diff
@@ -0,0 +1,322 @@
+# Spec: 99 Repository Tooling Contract - (no tool)
```

Comment:
https://github.com/Azure/azure-sdk-tools/blob/main/tools/azsdk-cli/docs/per-language.md 

Should we update that doc or incorporate them?